motor/frameworks/asyncio/__init__.py

```diff
@@ -97,12 +97,12 @@ def copy(_future):
         else:
             chained.set_result(return_value)
 
-    future._future.add_done_callback(functools.partial(loop.add_callback, copy))
```

Comment:
I had been a bit confused by this back when I was working on the PR to drop older Pythons. can you clarify what is `future._future` and why we don't need it anymore?

Reply:
This was and still is totally broken and completely untested. I'll open a new ticket for it.

Reply:
IIRC `to_list` breaks without this method. I tried to remove it when I was updating the framework files but couldn't figure out a way to get rid of the damn thing.

Reply:
I'll fix this in https://jira.mongodb.org/browse/MOTOR-600.